sunpy/coordinates/representation.py

```diff
@@ -117,3 +120,7 @@ class SphericalWrap180Representation(SphericalRepresentation):
 
     _unitrep = UnitSphericalWrap180Representation
 
+
+with warnings.catch_warnings():
+  warnings.simplefilter('ignore', AstropyDeprecationWarning)
+
```

Comment:
I am not sure that this is the correct way. But it isn't the right place in the script for this.

Reply:
Hey @nabobalis , Mistakenly I was testing on older version of Astropy, that why I was not getting a warning on testing , hopefully now I fixed the problem .